src/__generated__/About_artist.graphql.ts

```diff
@@ -0,0 +1,106 @@
+/* tslint:disable */
+
+import { ConcreteFragment } from "relay-runtime";
+export type About_artist = {
+    readonly has_metadata: boolean | null;
+    readonly is_display_auction_link: boolean | null;
```

Comment:
In [our schema](https://github.com/artsy/emission/blob/5126e093809c015f6ff2d5de867807505703a2cf/data/schema.graphql#L356-L357) this looks like:

```graphql
type ArtistItem implements Node {
  [...]
  # Only specific Artists should show a link to auction results.
  is_display_auction_link: Boolean
}
```

It would be nice to bring these comments into the interface in v1.1 of the TS generator

Reply:
Great idea!

Reply:
Great idea, I have some experience with this in some internal tooling we use where I work. Can you create an issue in https://github.com/relay-tools/relay-compiler-language-typescript ?

Reply:
https://github.com/relay-tools/relay-compiler-language-typescript/issues/35

Reply:
Ah great, I threw a few minutes at understanding the code to add it- happy to leave that to you 🍡

Reply:
Unless @kastermester has time immediately, I think that somebody else –such as yourself– contributing it would be great. At least you’ll know who to ask questions and perhaps @kastermester knows of some things to keep in mind from the top of his head.

Reply:
Ergh, that alloy, always pushing for more contributors 📟 

Will continue my digging

Reply:
The biggest concern here comes from interface types and how they interact with the type generation (without remembering exactly how that works in the TS plugin here mind you). If the type ends up being something like:

```ts
type MyType = {
  __typename: 'MyType1';
  name: string;
} | {
  __typename: 'MyType2';
  name: string;
} | {
  // Some comment here telling that this is to cover future types being added
  __typename: ' %other';
}
```

The question now becomes - which comment should be added to the `name` property - assuming there's one for the interface, and one for each of the types. Previously I have added the comment from the concrete types, such that the interface comment never shows up, but simply the concrete type comments. This has the effect on editors (at least VS Code but AFAIk all editors get their info from the TS language server) that you get the combined comment when simply accessing `name`.

I would love to find a better way to handle this sort of thing - but disregarding that, it should be pretty straight forward. If you have the time to make a pull request that'd be great.

Reply:
@orta FYI https://github.com/graphql/graphql-js/pull/1165